frontend/src/App.tsx

```diff
@@ -30,12 +30,32 @@ import ChannelNewPage from 'pages/ChannelNewPage';
 import ChannelPage from 'pages/ChannelPage';
 import { getWhoami, getHealthCheck } from 'api/api.v1';
 
-export const routes = (isLoggedin: boolean) => [
+export const routes = (isLoggedin: boolean, hasNickname: boolean) => [
```

Comment:
```suggestion
export const routes = (isLoggedin: boolean, isOnboarding: boolean) => [
```

함수 파라미터 이름으로는 '온보딩을 완료함'이 나을거 같아요. 호출할 때는 hasNickname으로 유지해서 닉네임을 가짐 === 온보딩을 완료함으로 읽을 수 있게..!

isOnboarding? didComplatedOnboarding?

Reply:
line 36, 50, 97

Reply:
isOnboarded로 하게씀다